### PR TITLE
[All] Debian buster support

### PR DESCRIPTION
**Pull request by nervo**
1. Add buster entries

*CHANGELOG.md*
```
### Added
- Debian buster support
```
*Makefile*
```
ROLE_DISTRIBUTIONS = [...] debian.buster
```
*meta/main.yml*
```
  platforms:
    - name: Debian
      versions:
        [...]
        - buster
```
4. Lint & Test
```
make lint.debian.buster
make test.debian.buster
```

- [x] manala.accounts
- [x] manala.alternatives
- [x] manala.ansible_galaxy
- [x] manala.apparmor
- [x] manala.apt
- [x] manala.aptly
- [x] manala.beanstalkd
- [x] manala.bind
- [x] manala.cloud_init
- [x] manala.composer
- [x] manala.cron
- [x] manala.deploy
- [x] manala.dhcp
- [x] manala.dnsmasq
- [x] manala.docker
- [x] manala.elasticsearch
- [x] manala.environment
- [x] manala.fail2ban
- [x] manala.files
- [x] manala.git
- [x] manala.gitlab
- [x] manala.glusterfs
- [x] manala.grafana
- [x] manala.haproxy
- [x] manala.influxdb
- [x] manala.java
- [x] manala.kernel
- [x] manala.locales
- [x] manala.logentries
- [x] manala.logrotate
- [x] manala.make
- [x] manala.maxscale
- [x] manala.merge
- [x] manala.mongodb _[TEMPLATES NEED UPDATE]_
- [x] manala.motd
- [x] manala.mount
- [x] manala.mysql _[30/08/19 : GALERA-3 & WSREP PACKAGES UNAVAILABLE YET]_ (http://releases.galeracluster.com/galera-3/)
- [x] manala.network
- [x] manala.nginx
- [x] manala.nodejs
- [x] manala.npm
- [x] manala.ntp
- [x] manala.ohmyzsh
- [x] manala.php
- [x] manala.postgresql
- [x] manala.proftpd
- [x] manala.proxmox
- [x] manala.redis
- [x] manala.rsyslog
- [x] manala.sensu _[PACKAGE UNAVAILABLE, go sensu go ¯\_(ツ)_/¯ ]_
- [x] manala.shorewall
- [x] manala.skeleton
- [x] manala.sqlite
- [x] manala.ssh
- [x] manala.sudo
- [x] manala.systemd
- [x] manala.telegraf
- [x] manala.timezone
- [x] manala.varnish
- [x] manala.vim
- [x] manala.yarn
- [x] manala.zsh

## Roles with Debian.manala.io dependancy

- [x] manala.ansible
- [x] manala.backup_manager
- [x] manala.heka _[DEPRECATED]_
- [x] manala.hugo
- [x] manala.keepalived
- [x] manala.mailhog
- [x] manala.mongo_express
- [x] manala.ngrok
- [x] manala.oauth2_proxy
- [x] manala.opcache_dashboard
- [x] manala.pam_ssh_agent_auth
- [x] manala.phantomjs
- [x] manala.phpmyadmin
- [x] manala.phppgadmin
- [x] manala.phpredisadmin
- [x] manala.rtail _[DEPRECATED]_
- [x] manala.supervisor
- [x] manala.thumbor
- [x] manala.vault

### CODE REVIEW

File: manala.ohmyzsh/meta/main.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ galaxy_info:
         - wheezy
         - jessie
         - stretch
+        - buster
   galaxy_tags:
     - system
-    - oh-my-zsh
+    - ohmyzsh
```
Comment (hyvs) [inline manala.ohmyzsh/meta/main.yml]: Official package name is `Oh-My-Zsh`, keep `oh-my-zsh` tag

File: manala.ssh/tasks/config.yml [hunk 2]
```diff
@@ -2,13 +2,19 @@
 
 - name: config > Template
   template:
-    src:  "{{ manala_ssh_config_template|ternary(manala_ssh_config_template, 'config/' ~ ansible_distribution|lower ~ '_' ~ ansible_distribution_release ~ '.j2') }}"
+    src:  "{{ manala_ssh_config_template|ternary(
```
Comment (hyvs) [inline manala.ssh/tasks/config.yml]: Should be : 
```
src: "{{ manala_ssh_config_template|ternary(
  manala_ssh_config_template,
  'config/' ~ ansible_distribution|lower ~ '_' ~ ansible_distribution_release ~ '.j2'
)}}"
```

File: manala.supervisor/tests/0100_install.goss.yml [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
   dpkg -s supervisor:
     exit-status: 0
     stdout:
       - "/^Version: [\\d\\.-]+manala\\d+~{{ .Env.DISTRIBUTION_RELEASE }}\\d+$/"
-  supervisord --version:
-    exit-status: 0
+{{ else }}
+      - "Version:"
+{{ end }}
+package:
```
Comment (hyvs) [inline manala.supervisor/tests/0100_install.goss.yml]: You should keep package present assert first

File: manala.yarn/CHANGELOG.md [hunk 4]
```diff
@@ -5,8 +5,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+## Added
+- Debian buster support
 
 ## [1.0.3] - 2018-06-05
+### Changed
```
Comment (hyvs) [inline manala.yarn/CHANGELOG.md]: You have two `changed` blocs

File: manala.keepalived/meta/main.yml [hunk 5]
```diff
@@ -14,9 +14,9 @@ galaxy_info:
   platforms:
     - name: Debian
       versions:
-        - wheezy
```
[the remark this answers is not in the document]
Comment (greedybro) [reply manala.keepalived/meta/main.yml]: On a jamais supporté wheezy sur ce role

File: manala.ohmyzsh/meta/main.yml (hunk 1 above)
Comment (greedybro) [reply manala.ohmyzsh/meta/main.yml]: Dashes are no longer authorized in tags, consequence of hyphen's gate

File: manala.supervisor/tests/0100_install.goss.yml (hunk 3 above)
Comment (greedybro) [reply manala.supervisor/tests/0100_install.goss.yml]: yes, and put condition on the right place =P

File: manala.docker/tests/0100_install.yml [hunk 6]
```diff
@@ -7,9 +7,6 @@
     manala_docker_config_daemon:
       - storage-driver: vfs
   pre_tasks:
-    - import_tasks: pre_tasks/disable_systemd.yml
```
[the remark this answers is not in the document]
Comment (greedybro) [reply manala.docker/tests/0100_install.yml]: Not needed anymore, docker inits itself with sysV correctly

**Pull request merged**